File: DarkOrbit 7.5.3/Net/netty/requests/QualitySettingsRequest.cs
using Ow.Utils;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ow.Net.netty.requests
{
    class QualitySettingsRequest
    {
        public const short ID = 26693;

        public short qualityEngine = 0;
        public short qualityEffect = 0;
        public bool qualityCustomized = false;
        public short qualityCollectable = 0;
        public short qualityPoizone = 0;
        public short qualityPresetting = 0;
        public short qualityBackground = 0;
        public short qualityAttack = 0;
        public short qualityExplosion = 0;
        public short qualityShip = 0;

        public void readCommand(byte[] bytes)
        {
            var param1 = new ByteParser(bytes);
            this.qualityAttack = param1.readShort();
            this.qualityBackground = param1.readShort();
            this.qualityPresetting = param1.readShort();
            this.qualityCustomized = param1.readBoolean();
            this.qualityPoizone = param1.readShort();
            this.qualityShip = param1.readShort();
            this.qualityEngine = param1.readShort();
            this.qualityExplosion = param1.readShort();
            this.qualityCollectable = param1.readShort();
            this.qualityEffect = param1.readShort();
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/clientCommands/QualitySettingsRequest.java
package simulator.netty.clientCommands;

import java.io.DataInputStream;
import java.io.IOException;

import simulator.netty.ClientCommand;

public class QualitySettingsRequest
        extends ClientCommand {

    public static final int     ID                 = 32602;
    public              int     qualityEngine      = 0;
    public              int     qualityEffect      = 0;
    public              boolean qualityCustomized  = false;
    public              int     qualityCollectable = 0;
    public              int     qualityPoizone     = 0;
    public              int     qualityPresetting  = 0;
    public              int     qualityBackground  = 0;
    public              int     qualityAttack      = 0;
    public              int     qualityExplosion   = 0;
    public              int     qualityShip        = 0;

    public QualitySettingsRequest(DataInputStream pIn) {
        super(pIn, ID);
    }

    public void read() {
        try {
            this.qualityBackground = in.readShort();
            this.qualityAttack = in.readShort();
            this.qualityPoizone = in.readShort();
            this.qualityCustomized = in.readBoolean();
            this.qualityEngine = in.readShort();
            this.qualityPresetting = in.readShort();
            this.qualityEffect = in.readShort();
            this.qualityCollectable = in.readShort();
            this.qualityShip = in.readShort();
            this.qualityExplosion = in.readShort();
            in.readShort();
        } catch (IOException e) {
        }
    }
}